tst/testbugfix/2017-08-03-StringFilterSetter.tst

```diff
@@ -5,4 +5,4 @@
 gap> IS_MUTABLE_OBJ;
 <Category "IsMutable">
 gap> Setter(IS_MUTABLE_OBJ);
-<Category "<<filter-setter>>">
+<Filter "<<filter-setter>>">
```

Comment:
Short story: The diff was caused by a bug that I introduced in `ClassOfOperation`. 
Both `Category` and `Filter` are wrong though: a setter is neither a filter nor a category; I made a fix that detects setters and gives `Setter` as the class of the operation.